miniapps/meshing/mesh-quality.cpp

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) 2010-2023, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+//
+//            --------------------------------------------------
+//                          Mesh Quality Miniapp
+//            --------------------------------------------------
+//
```

Comment:
Isn't it better to have this as part of `mesh-explorer`? Seems to have similar purpose.